tests/integration_tests/test_examples.py

```diff
@@ -119,3 +120,25 @@ def test_step_dm_control_env():
     assert subprocess.run(
         [EXAMPLES_ROOT_DIR / 'step_dm_control_env.py', '--n_steps', '1'],
         check=False).returncode == 0
+
+
+@pytest.mark.no_cover
```

Comment:
is it possible to provide default args in your examples so that the special cases here aren't necessary?

the general idea in this test is that most examples should be self-contained and can be covered by `test_algo_examples`

Reply:
I have to reduce epochs and rollouts per task to 1 to save time.